style.css

```diff
@@ -0,0 +1,188 @@
+body { font-family: "3rd party font", serif; }
+
+.Mobile_background {
+  width: 365px;
+  height: 5410px;
+  left: 0;
+  background: #f7f7f9;
+}
+
+header {
+  position: static;
+  width: 375px;
+  height: 92px;
+  left: 0;
+  top: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  position: static;
+  width: 76px;
+  height: 20px;
+  margin-top: 10px;
+  margin-left: 12px;
+}
+
+.menu {
+  position: static;
+  align-items: flex-start;
+  padding: 8px;
+  width: 40px;
+  height: 40px;
+  border-radius: 8px;
+}
+
+.headline_section .ellipse_11 {
+  position: absolute;
```

Comment:
- [ ] [Optional] I strongly suggest you avoid using `position: absolute`. Working in this way will tear apart your responsive design